DiscordBotApis/Data/GuildManager.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DiscordBotApis.Models;
+
+namespace DiscordBotApis.Data
+{
+    public class GuildManager : IGuildRepository
```

Comment:
I would prefer if we followed the naming convention of `Thing`: `IThing`, so this class would be `GuildRepository`

Reply:
fixed